drivers/usb/device/usb_dc_stm32.c

```diff
@@ -533,7 +533,7 @@ int usb_dc_ep_start_read(u8_t ep, u8_t *data, u32_t max_data_len)
 
 int usb_dc_ep_get_read_count(u8_t ep, u32_t *read_bytes)
 {
-	if (!EP_IS_OUT(ep)) {
+	if (!EP_IS_OUT(ep) || !read_bytes) {
```

Comment:
Not sure we need to check `read_bytes`.

Reply:
Right now, we don't but we should better keep the check to avoid future surprises if someone updates the driver and calls usb_dc_ep_get_read_count() with wrong inputs.